test/e2e/power-user/power-user.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { generateWalletState } from '../../../app/scripts/fixtures/generate-wallet-state';
+import { withFixtures } from '../helpers';
+import { loginWithBalanceValidation } from '../page-objects/flows/login.flow';
+import AccountListPage from '../page-objects/pages/account-list-page';
+import HeaderNavbar from '../page-objects/pages/header-navbar';
+import { Driver } from '../webdriver/driver';
+
+const withState = {
+  withAccounts: 30,
+  withConfirmedTransactions: 40,
+  withContacts: 40,
+  withErc20Tokens: true,
+  withNetworks: true,
+  withPreferences: true,
+  withUnreadNotifications: 15,
+};
+
+describe('Power user persona', function () {
+  it('loads the requested number of accounts', async function () {
+    await withFixtures(
+      {
+        title: this.test?.fullTitle(),
+        fixtures: (await generateWalletState(withState, true)).build(),
+        manifestFlags: {
+          testing: { disableSync: true },
```

Comment:
:+1: 
nice approach! There is another case where migration over rides a fixture state and we performed the steps manually in the specs, but I think I'll follow this example, to also fix that :heart_eyes: I'll share the PR in a bit!! thank you v much!